### PR TITLE
Wrap received teambuffs in an Accordion

### PR DESCRIPTION
**Pull request by nguyentvan7**
## Describe your changes

Wrap received team buffs in Accordion. When the buff list was long, the UI looked extremely cluttered

## Issue or discord link

- https://discord.com/channels/785153694478893126/785176548428087306/1218800800801882122

## Testing/validation

image
image

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: apps/frontend/src/app/PageTeam/TeamSetting/TeamComponents.tsx [hunk 1]
```diff
@@ -66,27 +70,42 @@ export function TeamBuffDisplay() {
...
+        bgcolor: theme.palette.contentLight.main,
+        borderRadius: '4px',
+        '&:before': {
+          display: 'none',
+        },
+      })}
+      disableGutters
+    >
+      <AccordionSummary sx={{ py: 1 }} expandIcon={<ExpandMoreIcon />}>
         <Typography>Received Team Buffs</Typography>
```
Comment (frzyc) [inline apps/frontend/src/app/PageTeam/TeamSetting/TeamComponents.tsx:84]: Should we add a tally here?

**Comment by frzyc** [comment]
Also, I think the "golden solution" here is just to condense the elemental res shred into a single buff. (through we'd have to deal with in this case, an additional geo shred)

**Comment by JohnnyToumieh** [comment]
> Also, I think the "golden solution" here is just to condense the elemental res shred into a single buff. (through we'd have to deal with in this case, an additional geo shred)

If you were to do this, don't forget about Res Buff (through Ning's E or 4 different elements resonance)

**Comment by nguyentvan7** [comment]
Condensing elemental res shred is just a bandaid, not a golden solution. The list can still get very long without it. Also condensing them is inaccurate to in-game

**Review by frzyc — APPROVED** [review]

**Pull request merged**